src/LibraryManager.Vsix/Contracts/Logger.cs

```diff
@@ -178,5 +179,147 @@ private static void LogOperationSummary(IEnumerable<ILibraryOperationResult> tot
                 LogEvent(messageText, LogLevel.Operation);
             }
         }
+
+        public static void LogErrors(IEnumerable<ILibraryOperationResult> results)
+        {
+            foreach (ILibraryOperationResult result in results)
+            {
+                if (result.Errors.Any())
+                {
+                    foreach (IError error in result.Errors)
+                    {
+                        LogEvent(error.Message, LogLevel.Operation);
+                    }
+                }
+            }
+        }
+
+        private static string GetLibraryId(IEnumerable<ILibraryOperationResult> totalResults, OperationType operation)
```

Comment:
>GetLibraryId [](start = 30, length = 12)

Prafull moved this helper method and the ones below it somewhere else. Make sure you don't add them back.